website/docs/api/ref/PrivateChannel.md

```diff
@@ -51,6 +51,35 @@ interface IPrivateChannel : IChannel, IIntentResult
 }
 ```
 
+</TabItem>
+<TabItem value="golang" label="Go">
+
+```go
+type PrivateChannel struct {
+    Channel
+}
+
+func (privateChannel *PrivateChannel) AddEventListener(eventType *PrivateChannelEventTypes, handler EventHandler) <-Result[Listener] {
+    // Implementation here
+}
+func (privateChannel *PrivateChannel) Disconnect() <-Result[any]  {
+    // Implementation here
+}
+
+// Deprecated
+func (ch *PrivateChannel) OnAddContextListener(handler func(contextType *ContextType)) Result[Listener] {
+    // Implementation here
+}
+
+func (ch *PrivateChannel) OnUnsubscribe(handler func(contextType *ContextType)) Result[Listener] {
+    // Implementation here
+}
+
+func (ch *PrivateChannel) OnDisconnect(handler func()) Result[Listener] {
+    // Implementation here
+}
```

Comment:
Do you want to include the deprecated functions? If not we can lose them:

```suggestion
```

Reply:
> Do you want to include the deprecated functions? If not we can lose them:

I was not sure if I should add them here in case someone wants to implement the older version, but you are right and if someone wants to, they can extrapolate, rather than us keeping it in the standard